samples/net/sockets/tls_configurations/sample.yaml

```diff
@@ -0,0 +1,41 @@
+common:
+  tags:
+    - mbedtls
+    - socket
+  platform_allow:
+    - native_sim
+    - native_sim/native/64
+  # Testing requires the OpenSSL to be started, but this is currently not
+  # available on the CI so we skip it. Testing is possible locally though.
+  skip: true
```

Comment:
I would like to point out that this is a sample for now, but [I would like to make a test](https://github.com/zephyrproject-rtos/zephyr/pull/77567#issuecomment-2320592981) of it. So `skip` here is temporary to avoid CI failures, but this is going to be removed once I figure out how to test it properly.

Reply:
So if the ultimate goal to make a TLS testsuite of it, sholdn't this be in `tests/net` instead of `samples/net`? I wouldn't mind limiting the functionality to a single platform only then.

Reply:
Yes, I am planning to move it to `test` once I find the proper way to test it ;)